src/main/resources/text/helpForAllCommands.txt

```diff
@@ -7,6 +7,12 @@ contact list                Lists all contacts
 contact find                Find contacts given search keyword(s)                       contact find KEYWORD [MORE_KEYWORDS]
 contact delete              Deletes a contact                                           contact delete INDEX
 contact sort                Sorts contact list                                          contact sort KEYWORD [ORDER]
+contact archive             Sends a contact to the archive                              contact archive INDEX
+
+Archive commands
+----------------------
```

Comment:
I think naming contact archive to archive add or archive would be better than separating an archive functionality into two different naming conventions

Reply:
Will go for archive add.